client/lib/analytics/ad-tracking/record-order.js

```diff
@@ -79,6 +79,7 @@ export async function recordOrder( cart, orderId ) {
 	recordOrderInCriteo( cart, orderId );
 	recordOrderInGAEnhancedEcommerce( cart, orderId, wpcomJetpackCartInfo );
 	recordOrderInJetpackGA( cart, orderId, wpcomJetpackCartInfo );
```

Comment:
Since the function below is called `recordOrderInWPcomGA4`, for consistency sake should we also rename this to `recordOrderInJetpackGA4 `?

Reply:
I agree! :)